### ISSUE TITLE
Add 503 to Backoff Status in Src/Store/Lookup.ts, to Defeat Zotac 503 Page & Captcha

### ISSUE THREAD

**Issue by MattTheTechLV** [jef/nvidia-snatcher#343]
### Description

Since Zotac Recently updated their site, now when polling you get a looping 503 error, because it sits constantly on the "Checking Your Browser before accessing Zotac.com" screen as shown below.

![Zotac 503 Error](https://user-images.githubusercontent.com/61740449/94363501-61fcd580-0077-11eb-912a-f2bf25c8c6fc.jpg)

Because 503 isn't recognized as an error, the site just constantly loads every 2 seconds, and you are never able to get through to the website to check stock. (As seen Below)

![Console-Zotac-503](https://user-images.githubusercontent.com/61740449/94363613-3d552d80-0078-11eb-95cd-b6b2ba605324.jpg)

"PAGE_TIMEOUT" and "PAGE_SLEEP" Variables have no effect on this behavior, no matter how high they are set, the site still refreshes every 2 seconds, and you get stuck in the 503 Loop.

### Possible solution

In order to deal with this, all I had to do was update the "Lookup.ts" file inside of "Src/Store/" to recognize the 503 Error and to switch into backoff mode. Then as long as you have a maximum of 10000ms or so for your "PAGE_BACKOFF_MAX", after a few 503 errors it will successfully get out of that loop, and actually start checking stock. When Running in Headless Mode Off I observed that Zotac's site will bring up a captcha after waiting a few seconds, but simply refreshing appears to beat the captcha, so the "backoff" entry is all that seems to be needed at this time. 

This is only my 2nd submission to this project, so I am sorry if I am not formatting this feature submission correctly. I do hope this is helpful though, as it worked for me. Here is the Code I input starting at line 87 and ending at line 93:

86
87`	if (response?.status() === 503) {`
88`		Logger.warn(Print.backoff(link, store, backoff.time, true));`
89`		await delay(backoff.time);`
90`		backoff.count++;`
91`		backoff.time = Math.min(backoff.time * 2, Config.browser.maxBackoff);`
92`		return;`
93`	}`
94

![Lookup Code Added](https://user-images.githubusercontent.com/61740449/94363761-704bf100-0079-11eb-9335-91f261d8e832.jpg)

**Comment by andrewmackrodt** [comment]
Thanks for the contribution @MattTheTechLV. I've tested your suggestions and they are working for me. This should be fixed in #340.

As an aside, `PAGE_SLEEP` works on a per link basis versus per store which is why it looks like requests are made every 2 seconds. The `trinity oc` is checked immediately after `trinity` without any pause but then there is a ~5 second delay between the next "batch" of calls. Additionally, the page is closed immediately after we check if there is stock.

`PAGE_BACKOFF` applies for the entire store and unintentionally leaves the page open until the delay period has passed. In the case of the Zotac store, this behaviour is what allows us to pass CloudFlare's check so I've made sure to add a comment that this behaviour is now intended!
